src/EscapeAnalysis.jl

```diff
@@ -216,6 +217,20 @@ function find_escapes(ir::IRCode, nargs::Int)
                         continue
                     elseif ft === typeof(isa) || ft === typeof(typeof) || ft === typeof(Core.sizeof)
                         continue
+                    elseif ft === typeof(ifelse) && length(stmt.args) === 4
```

Comment:
I don't know if `length(stmt.args) == 4` would be faster than this and if this check has already done before our analysis (i.e. any `ifelse` call without 4 args will fail)

Reply:
Good point. I think we can remove this kind of check once we tackle #6 (, where we will assert a call won't throw at runtime ahead of this check, and it will have filtered out those malformed builtin calls).
For now, I'd like to preserve this since otherwise we may encounter `BoundsError` in the next line.